cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,111 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
+  const workerName = 'Vega';
+  const workerDepartment = 'Sales';
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.visit('/webtables');
+    });
+  });
+
+  it('should check a pagination', () => {
+    for (let i = 0; i < 11; i++) {
+      cy.task('generateUser').then((user) => {
+        cy.addNewWorker(user);
+      });
+    }
+    cy.contains('.-btn', 'Next').click();
+    cy.get('[aria-label="jump to page"]').should('have.value', '2');
+    cy.contains('.-btn', 'Previous').click();
+    cy.get('[aria-label="jump to page"]').should('have.value', '1');
+  });
+
+  it('should check a rows count selection', () => {
+    cy.get('[aria-label="rows per page"]').select('25');
+    cy.get('.rt-tbody .rt-tr').should('have.length', 25);
+  });
+
+  it('should add a new worker', () => {
+    cy.addNewWorker(user);
+    cy.get('.rt-table').should('contain', user.lastName);
+  });
+
+  it('should delete a worker', () => {
+    cy.deleteWorker();
+    cy.get('.rt-table').should('not.contain', user.lastName);
```

Comment:
```suggestion
    cy.get('.rt-table').should('not.contain', user.lastName);
```
table will not contain this user because it hasn't been added before

Reply:
so this test will be passed always